semcore/inline-input/src/index.d.ts

```diff
@@ -55,7 +55,7 @@ export type InlineInputProps = BoxProps & {
    * defines callback (`onCancel` or `onConfirm`) triggered when `blur` event out of container fired
    * Triggered after all previous macrotasks completed (internally called inside of `setTimeout`)
    */
-  onBlurBehavior?: 'cancel' | 'confirm';
+  onBlurBehavior?: 'cancel' | 'confirm' | 'none';
```

Comment:
what is the difference between `none` and `undefined`?

Reply:
When they set `undefined` the default value `confirm` is used